### PR TITLE
fixes open ai example

### PR DESCRIPTION
**Pull request by sehz**
added `serge-json` as dependency

### CODE REVIEW

File: connectors.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ http:
 Copy the configuration to a file and star the connector on your local machine:
 
 ```bash
-cdk deploy start --ipkg infinyon-http-source-0.3.3.ipkg -c quotes-periodic.yaml
+cdk deploy start --ipkg infinyon-http-source-0.6.ipkg -c quotes-periodic.yaml
```
Comment (ajhunyady) [inline connectors.md:44]: should be 0.3.6

File: connectors.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ http:
 Copy the configuration to a file and star the connector on your local machine:
 
 ```bash
-cdk deploy start --ipkg infinyon-http-source-0.3.3.ipkg -c quotes-stream.yaml
+cdk deploy start --ipkg infinyon-http-source-0.6.ipkg -c quotes-stream.yaml
```
Comment (ajhunyady) [inline connectors.md:81]: same

**Review by ajhunyady — COMMENTED** [review]
Should have branced from `preview-8` instead of `master`, otherwise you'll get lost of merge issues.

**Comment by sehz** [comment]
this is wrong one. closing this

**Pull request closed without merging**